package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.1.32",
+  "version": "0.2.0",
```

Comment:
@Oniel1721 why go to 0.2 and not 0.1.33 ?

Reply:
because is not a patch is a minor change

Reply:
patches version are for bug fixes 
minor version are for Backward compatible new features
check on npm semantic versioning docs https://docs.npmjs.com/about-semantic-versioning